js/digifinex.js

```diff
@@ -1445,6 +1451,60 @@ module.exports = class digifinex extends Exchange {
         };
     }
 
+    parseTransfer (transfer, currency = undefined) {
+        //
+        //     {
+        //         "code": 0
```

Comment:
We should parse it with parseTransferStatus (see mentions of it in the other exchanges).